dialects/postgresql/src/main/kotlin/app/cash/sqldelight/dialects/postgresql/grammar/PostgreSql.bnf

```diff
@@ -249,6 +250,8 @@ tsvector_data_type ::= 'TSVECTOR'
 
 xml_data_type ::= 'XML'
 
+geometry_data_type ::= 'GEOMETRY' [ LP 'Point' COMMA {signed_number} RP ]
```

Comment:
This is quite basic currently, there are many more options: https://www.postgresql.org/docs/current/datatype-geometric.html